AEM/intel-extension-for-openxla.md

```diff
@@ -0,0 +1,249 @@
+## Intel® Extension for OpenXLA\*
+
+OpenXLA is an open-source Machine Learning compiler ecosystem co-developed by AI/ML industry leaders that lets developers compile and optimize models from all popular ML frameworks on a wide variety of hardware. We are pleased to announce Intel® Extension for OpenXLA\*, which seamlessly runs AI/ML models on Intel GPUs. Intel® Extension for OpenXLA\* is a high-performance deep learning extension implementing the [OpenXLA](https://github.com/openxla/xla) PJRT C API (see [RFC](https://github.com/openxla/community/blob/main/rfcs/20230123-pjrt-plugin.md) for details), allowing multiple AI frameworks to compile StableHLO, an operation set for high-level operations for numeric computation, as well as dispatching the executable to Intel GPUs. 
+
+<br><div align=center><img src="/content/dam/developer/articles/technical/accelerate-stable-diffusion-on-intel-gpus-with-intel-extension-for-openxla/Intel-Data-Center-GPU-Max-Series-1100-PCIe-Card.png"></div>
+
+<p align="center">Figure 1: Intel® Data Center GPU Max Series 1100 PCIe Card</p>
+
+The PJRT plugin for Intel GPU is based on LLVM + SPIR-V IR code-gen technique.  It integrates with optimizations in oneAPI-powered libraries, such as Intel® oneAPI Deep Neural Network Library (oneDNN) and Intel® oneAPI Math Kernel Library (oneMKL). With the optimizations on linear algebra, operator fusion, layout propagation, etc., developers can speed up their workloads without any device-specific codes. JAX is the first supported front-end. Intel® Extension for OpenXLA\* enables and accelerates training and inference of different scales workloads, including Large Language Models (LLMs) or multi-modal models on Intel GPUs, such as Stable Diffusion. 
+
+<br><div align=center><img src="/content/dam/developer/articles/technical/accelerate-stable-diffusion-on-intel-gpus-with-intel-extension-for-openxla/Intel-Extension-for-OpenXLA-Architecture%20.png"></div>
+
+<p align="center">Figure 2: Intel® Extension for OpenXLA* Architecture</p>
+
+## Enabling Intel® Extension for OpenXLA\* 
+
+In this section, we show how to enable JAX applications on Intel GPUs with Intel® Extension for OpenXLA\*.
+
+### Installation
```

Comment:
possible also put a reference link to official Github repo page or github.io doc if any?

Reply:
Thanks. Done.